go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -598,16 +600,29 @@ func egressGetL4Match(ports []egressfirewallapi.EgressFirewallPort) string {
 	return fmt.Sprintf("(%s)", l4Match)
 }
 
-func getClusterSubnetsExclusion() string {
+func getClusterSubnetsExclusion(destination matchTarget) string {
+	if destination.kind == matchKindV4AddressSet || destination.kind == matchKindV6AddressSet {
```

Comment:
@JacobTanenbaum : `destination.kind == *AdresSet` means it's been computed from a DNS name. Could you confirm if DNS for egress firewall only supports DNS names which resolve to IPs external to the cluster? I.e: we don't support internal cluster DNS names here right?

Reply:
I'll bet we don't make a distinction for the DNS lookup... but it's *Egress* firewall and is meant to control traffic crossing the cluster boundary. Plus I hope we prevent cluster-internal addresses from being used in the EgressFirewall non-DNS parts. Based on that I think we can ignore internal cluster names even if users try to use them.